src/rf/js/src/home/components/library.js

```diff
@@ -543,7 +554,7 @@ var ImageMetadata = React.createBackboneClass({
                 <div className="layer-detail-content">
                     <img className="img-preview" src={image.thumb_large || 'https://placehold.it/300x300'} />
                     <hr />
-                    <div className="scrollable">
+                    <div className="scrollable" style={{overflow: 'hidden'}}>
```

Comment:
In my testing, the parent div is still able to scroll vertically.

![image](https://cloud.githubusercontent.com/assets/43062/11131229/95280642-8957-11e5-85ad-2626d0c5b8f2.png)